pages/common/circo.md

```diff
@@ -0,0 +1,25 @@
+# circo
+
+> Render an image of a `circular` network graph from a `graphviz` file.
+> Layouts are `dot`, `neato`, `twopi`, `circo`, `fdp`, `sfdp`, `osage` & `patchwork`.
+> More information: <http://graphviz.org/doc/info/command.html>.
+
+- Render a `png` image with a filename based on the input filename and output format:
+
+`circo -Tpng -O {{path/to/input.gv}}`
+
+- Render a `svg` image with the specified output filename:
+
+`circo -Tsvg -o {{path/to/image.svg}} {{path/to/input.gv}}`
+
+- Render the output in `ps`, `pdf`, `svg`, `fig`, `png`, `gif`, `jpg`, `json`, or `dot` format:
+
+`circo -T{{format}} -O {{path/to/input.gv}}`
```

Comment:
Space missing here?

Reply:
@CleanMachine1 , No space between -T and format: -Tpng -Tsvg -Tpdf, etc. See the other sample lines.

Also, not "mass changes". "Multiple files", perhaps. These are new files. The `dot.md` file in the other PR is the only previously existing file. If you accept #6270, there will not be any changes to `dot.md` in this PR as it will match.

Reply:
The `mass changes` label is a bit confusing, but per definition it shuld be used when multiple pages are affected. In the end labels are nothnig strict, but just something to classify a PR